tests/cli/RunCest.php

```diff
@@ -190,6 +190,57 @@ public function runTestWithDataProviders(\CliGuy $I)
         $I->seeInShellOutput("OK");
     }
 
+    public function runTestWithDataProvidersFailure(\CliGuy $I)
+    {
+        $I->executeCommand('run tests/dummy/DataProvidersFailureTest.php --no-exit -v');
+        $I->seeInShellOutput('There was 1 warning');
+        $I->seeInShellOutput('PHPUnit_Framework_Warning');
+        $I->seeInShellOutput('The data provider specified for DataProvidersFailureTest::testIsTriangle is invalid.');
+        $I->seeInShellOutput('Method rectangle does not exist');
+        $I->seeInShellOutput("WARNINGS!");
+        $I->seeInShellOutput("Tests: 1, Assertions: 0, Warnings: 1.");
+    }
+
+    public function runTestWithDataProvidersExceptionStdout(\CliGuy $I)
+    {
+        $I->executeCommand('run tests/dummy/DataProvidersExceptionCest.php 2> /dev/null', false);
+        $I->dontSeeInShellOutput('There was 1 warning');
+        $I->dontSeeInShellOutput('PHPUnit_Framework_Warning');
+        $I->dontSeeInShellOutput('The data provider specified for DataProvidersExceptionTest::testIsTriangle is invalid.');
+        $I->dontSeeInShellOutput("WARNINGS!");
+        $I->dontSeeInShellOutput("OK");
+        $I->dontSeeInShellOutput("[Exception]");
+        $I->dontSeeInShellOutput("Something went wrong!!!");
```

Comment:
Actually - the output from the `executeCommand` strips even the 
```
Codeception PHP Testing Framework v2.3.4
Powered by PHPUnit 5.7.21 by Sebastian Bergmann and contributors.
```
headers - I've added a test to check that the output is blank and the result code is 1.